### ISSUE TITLE
Error creating bean awsSdkS3AmazonS3Service

### ISSUE THREAD

**Issue by pinkishpanda** [agorapulse/grails-aws-sdk-s3#3]
Hi,

I'm using org.grails.plugins:aws-sdk-s3:2.0.2, with awsJavaSdkVersion=1.11.41 on Grails 3.0.14, and I'm getting these errors when running the app:

ERROR grails.boot.GrailsApp - Application startup failed
org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'com.myapp.DocumentController': Initialization of bean failed; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'awsSdkS3AmazonS3Service': Cannot create inner bean '(inner bean)#214f498c' while setting bean property 'target'; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name '(inner bean)#214f498c': Invocation of init method failed; nested exception is java.lang.NoSuchMethodError: com.fasterxml.jackson.databind.ObjectMapper.enable([Lcom/fasterxml/jackson/core/JsonParser$Feature;)Lcom/fasterxml/jackson/databind/ObjectMapper;
        at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:547) ~[spring-beans-4.1.8.RELEASE.jar:4.1.8.RELEASE]
        at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:476) ~[spring-beans-4.1.8.RELEASE.jar:4.1.8.RELEASE]
        at org.springframework.beans.factory.support.AbstractBeanFactory$1.getObject(AbstractBeanFactory.java:303) ~[spring-beans-4.1.8.RELEASE.jar:4.1.8.RELEASE]

Do you have any suggestions of what could be wrong here?

**Comment by benorama** [comment]
It looks like a lib conflicts linked to _Jackson_:
`java.lang.NoSuchMethodError: com.fasterxml.jackson.databind.ObjectMapper.enable`

Have you tried to run `gradle dependency-report` to which dependencies your getting for `com.fasterxml.jackson` ?

**Comment by pinkishpanda** [comment]
Thanks for the prompt reply. I checked the dependencies, and found that `org.springframework.boot:spring-boot-starter-actuator` uses `com.fasterxml.jackson.core:jackson-databind:2.4.6`, while `org.grails.plugins:aws-sdk-s3` uses `2.6.6`. 

But after I excluded the older one in build.gradle, I'm still getting the same error. I've tried specifying the newer version explicitly in build.gradle. I even checked the jackson lib jars that were to be uploaded to the server, it's already `2.6.6`. But I still get the same error as above :(

Can there be any other possible cause? Or what's the best way to resolve this? Many thanks for your help.
